daemon/cmd/policy.go

```diff
@@ -452,20 +458,52 @@ func (d *Daemon) policyAdd(sourceRules policyAPI.Rules, opts *policy.AddOptions,
 // to a policy repository for a daemon. This currently consists of endpoint
 // regenerations / policy revision incrementing for a given endpoint.
 type PolicyReactionEvent struct {
-	wg                *sync.WaitGroup
-	epsToBumpRevision *policy.EndpointSet
-	endpointsToRegen  *policy.EndpointSet
-	newRev            uint64
-	upsertIdentities  map[string]*identity.Identity // deferred CIDR identity upserts, if any
-	releasePrefixes   []*net.IPNet                  // deferred CIDR identity deletes, if any
+	wg                  *sync.WaitGroup
+	epsToBumpRevision   *policy.EndpointSet
+	addendpointsToRegen *policy.EndpointSet
+	delendpointsToRegen *policy.EndpointSet
```

Comment:
Same as above, `deletedEndpointsToRegen` would be preferrable.

Reply:
Done